cf-custom-resources/lib/dns-delegation.js

```diff
@@ -264,28 +262,27 @@ const waitForRecordSetChange = function (route53, changeId) {
 
 exports.domainDelegationHandler = async function (event, context) {
   var responseData = {};
-  var physicalResourceId;
+  const props = event.ResourceProperties;
+  const physicalResourceId = props.SubdomainName;
```

Comment:
For `Delete`, it was `event.PhysicalResourceId`, do we guarantee somewhere that `props.SubdomainName` is the same as `event.PhysicalResourceId` when the "Delete" action is performed? I feel like in that case we should include one line of explanation here, maybe?